pep/app/pods/user/model.ts

```diff
@@ -84,6 +89,12 @@ export default class User extends DS.Model {
               })
             : [];
     }
+
+    get loggedInLabel() {
```

Comment:
@napafundi this would probably be better in the current user service. This is more a display item and the model shouldn't really deal with those.